actgen/agents/dqn.py

```diff
@@ -23,8 +23,8 @@ def __init__(self, observation_space, action_space, params):
         params = list(self.q.parameters())
         self.optimizer = torch.optim.Adam(params, lr=self.params['learning_rate'])
 
-    def save(self, is_best, seed, regularizer):
-        self.q.save('qnet' + f'_seed{seed}_' + regularizer, 'results/', is_best)
+    def save(self, is_best, fname, results_dir, tag_dir):
```

Comment:
Sorry for not being specific enough. I'd like just the filename and a single directory (plus `self` and `is_best`) to be the inputs to this function, rather than two directories that need to be combined.